package.json

```diff
@@ -11,7 +11,18 @@
   "configSchema": {
     "executablePath": {
       "type": "string",
-      "default": "codeclimate"
+      "default": "/usr/local/bin/codeclimate",
```

Comment:
How about instead we remove the existing check on the path, wrap the [call to exec](https://github.com/AtomLinter/linter-codeclimate/blob/d0c42f46717d19c2ed61b868eef0e676c3a4ebe5/lib/index.js#L173) in a `try`/`catch` block, and handle the `ENOENT` error if one is thrown?

It looks like this will potentially need to stop wrapping this in a call `/bin/bash`, I'm not sure of the implications of that though.